unity/Assets/Scripts/CanOpen_Object.cs

```diff
@@ -42,53 +41,35 @@ protected enum MovementType { Slide, Rotate, ScaleX, ScaleY, ScaleZ};
 	[SerializeField]
     protected MovementType movementType;
 
-	//keep a list of all objects that, if able to turn on/off, must be in the Off state before opening (no opening microwave unless it's off!);
+	// keep a list of all objects that, if able to turn on/off, must be in the Off state before opening (no opening microwave unless it's off!);
 	private List<SimObjType> MustBeOffToOpen = new List<SimObjType>()
 	{SimObjType.Microwave};
 
-    //these objects, when hitting another sim object, should reset their state because it would cause clipping. Specifically used
-    //for things like Laptops or Books that can open but are also pickupable and moveable. This should not include static
-    //things in the scene like cabinets or drawers that have fixed positions
+    // these objects, when hitting another sim object, should reset their state because it would cause clipping. Specifically used
+    // for things like Laptops or Books that can open but are also pickupable and moveable. This should not include static
+    // things in the scene like cabinets or drawers that have fixed positions
     // private List<SimObjType> ResetPositionIfPickupableAndOpenable = new List<SimObjType>()
     // {SimObjType.Book, SimObjType.Laptop};
 
 
     [Header("References for the Open or Closed bounding box for openable and pickupable objects")]
-    //the bounding box to use when this object is in the open state
+    // the bounding box to use when this object is in the open state
     [SerializeField]
     protected GameObject OpenBoundingBox;
 
-    //the bounding box to use when this object is in the closed state
+    // the bounding box to use when this object is in the closed state
     [SerializeField]
     protected GameObject ClosedBoundingBox;
 
-
-
-
-    #if UNITY_EDITOR
-    void OnEnable ()
-    {
-		//debug check for missing CanOpen property
-        if (!gameObject.GetComponent<SimObjPhysics>().DoesThisObjectHaveThisSecondaryProperty(SimObjSecondaryProperty.CanOpen))
-        {
-            Debug.LogError(gameObject.transform.name + " is missing the Secondary Property CanOpen!");
-        }
-    }
-    #endif
```

Comment:
No. This was just moved to an all-encompassing #if UNITY_EDITOR block, so that there aren't repeated UNITY_EDITOR blocks within the same context.

https://github.com/allenai/ai2thor/blob/openness/unity/Assets/Scripts/CanOpen_Object.cs#L102